qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy

```diff
@@ -313,12 +309,10 @@ splunk:
 
         // Verify the groups are created successfully, and specify the origin declarative.
         def expectedGroups = [VALID_DEFAULT_GROUP, VALID_DECLARATIVE_GROUP]
-        def foundGroups = verifyDeclarativeGroups(authProvider.getId(), expectedGroups)
-        assert foundGroups == expectedGroups.size() :
-                "expected to find ${expectedGroups.size()} groups, but only found ${foundGroups}"
+        verifyDeclarativeGroups(authProvider.getId(), expectedGroups)
 
-        def notifier = verifyDeclarativeNotifier(VALID_NOTIFIER)
-        assert notifier
+        // Verify the notifier is created successfully, and does specify the origin declarative.
+        verifyDeclarativeNotifier(VALID_NOTIFIER)
```

Comment:
Can we do the same for permission set, access scope and auth provider?

Reply:
we already do, or do you mean skipping the return value? I don't think we can since we need the IDs of perm set / access scope / auth provider for later verifying stuff.